.github/workflows/build-test-macos.yml

```diff
@@ -79,6 +89,16 @@ jobs:
           CMAKE_C_COMPILER: ${{ matrix.c-compiler }}
           MR_VERSION: ${{ inputs.app_version }}
           MR_USE_CPP_23: "ON"
+          MR_CMAKE_OPTIONS: -DMESHLIB_BUILD_MRMESH_PY_LEGACY=${{ fromJSON('["ON", "OFF"]')[inputs.mrbind] }}
+
+      - name: Generate and build MRBind bindings
+        if: ${{inputs.mrbind}}
+        env:
+          # Add updated Make from homebrew to PATH. Arm Macs use `/opt/homebrew`, while x86 Macs use `/usr/local`. Add both for simplicity.
+          PATH: /opt/homebrew/opt/make/libexec/gnubin:/usr/local/opt/make/libexec/gnubin:${{env.PATH}}
+        run: |
+          make --version
+          make -f scripts/mrbind/generate.mk -B --trace PYTHON_PKGCONF_NAME=python-3.10-embed EXTRA_CFLAGS='-DMB_PB11_ALLOW_STD_EXPECTED=0 -DMR_USE_STD_EXPECTED=0' MESHLIB_SHLIB_DIR=build/${{matrix.config}}/bin
```

Comment:
can we parametrize python version, or read it form somewhere? 
also, should all these flags be hidden in generate.mk, or higher level script?